README.md

```diff
@@ -1,7 +1,7 @@
-# portfolio-Mobile-Version
+# Mobile-Version
```

Comment:
- [ ] Let's give credit to [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) the author of the original design, as required by the [Creative Commons License](https://creativecommons.org/licenses/) through the "Acknowledgements" section of the Readme.